website/source/api/system/plugins-reload-backend.html.md

```diff
@@ -23,10 +23,10 @@ This endpoint reloads mounted plugin backends.
 
 ### Parameters
 
-- `plugin` `(string: "")` – The name of the plugin to reload, as 
+- `plugin` `(string: "")` – The name of the plugin to reload, as
   registered in the plugin catalog.
 
-- `mounts` `(slice: [])` – Array or comma-separated string mount paths 
```

Comment:
This was the _only_ occurrence of "slice" in the documentation, so I fixed it in this PR.